app-android-xamarin/Moodis/Moodis/Feature/SignIn/SignInActivity.cs

```diff
@@ -105,6 +105,8 @@ private void InitButtonsAndInputs()
                     if (SignInViewModel.Authenticate(usernameInput.Text, passwordInput.Text))
                     {
                         SetResult(Result.Ok);
+                        var cameraActivity = new Intent(this, typeof(CameraActivity));
```

Comment:
Remove this, since camera activity is started when result is sent to main activity